Examples/JS/flow/BoardType.js

```diff
@@ -9,13 +9,15 @@
 
 import type { PlankDate, PlankURI } from './runtime.flow.js';
 import type { ImageType } from './ImageType.js';
+import type { UserType } from './UserType.js';
 
 export type BoardType = $Shape<{|
   +name: ?string,
   +id: ?string,
-  +image: ?ImageType,
+  +image: ImageType,
```

Comment:
Why did this change? Seems not related to this change though.

Reply:
@maicki  I don't know exactly. This was all generated code.